e2e/scenario_helpers_test.go

```diff
@@ -261,6 +289,62 @@ func getExpectedPackageVersions(packageName, distro, release string) []string {
 	return expectedVersions
 }
 
+func Map[T, U any](ts []T, f func(T) U) []U {
```

Comment:
should this be in a more generic file ?

Reply:
I'll move it somewhere more generic - there is the toolkit package that seems appropriate.